api/app.js

```diff
@@ -1,11 +0,0 @@
-import express from "express";
```

Comment:
Did you delete this by accident? We should probably keep this file here, as once we merge to main @sharkand73 dummy data response will be returned.

Reply:
Ok, managed to bring back the file i deleted